must-gather/collection-scripts/pre-install.sh

```diff
@@ -43,13 +48,15 @@ deploy(){
 }
 
 labels(){
+    if [ -n "${storageClusterPresent}" ]; then
```

Comment:
```suggestion
    if [ -z "${storageClusterPresent}" ]; then
```

Reply:
I would be `-n` because we want to execute that part when the storagecluster is present`-z`.
-n is the opposite of -z. `if [ -n "${storageClusterPresent}" ]` passes if the string is not null and not empty.